src/test/java/ResponseBuilderTest.java

```diff
@@ -1,10 +1,23 @@
 import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Test;
 
 public class ResponseBuilderTest {
+
+    private ResponseBuilder responseBuilder;
+
+    @Before
+    public void setUp() {
+        responseBuilder = new ResponseBuilder();
+    }
     @Test
     public void buildResponseWithStatusCode200AndEmptyBody() {
-        ResponseBuilder responseBuilder = new ResponseBuilder();
         Assert.assertEquals("HTTP/1.1 200 OK\r\n\r\n", responseBuilder.okayWithEmptyBody());
     }
+
+    @Test
+    public void buildResponseWithStatusCode20AndNoBody() {
```

Comment:
There is a typo in `StatusCode20`, I think you mean `StatusCode200`.

What is the difference between `responseBuilder.okayWithEmptyBody()` and `responseBuilder.okayWithNoBody()`? To me, it looks like `responseBuilder.okayWithEmptyBody() == responseBuilder.okayWithNoBody()`

Reply:
They are the same, the difference is one has no body (`HEAD` request) and one has an empty body represented by an empty string (`GET` request) - I wasn't sure whether there needed to be a distinction between the two requests. If they are the same, then I guess I should remove the duplication!